common/unsafe/src/main/java/org/apache/spark/unsafe/Platform.java

```diff
@@ -319,4 +301,36 @@ public static void throwException(Throwable t) {
       DOUBLE_ARRAY_OFFSET = 0;
     }
   }
+
+  // This requires `majorVersion` and `_UNSAFE`.
```

Comment:
This is moved at the bottom of this file because `majorVersion` and `_UNSAFE` should be initialized before this.